src/features/login/LoginPage.js

```diff
@@ -1,32 +1,47 @@
 import React from "react";
 import "./LoginPage.css";
-import { useState } from "react";
+import { useState , useEffect } from "react";
 
 
 
 
 export default function LoginPage() {
 
+    const [country, setCountry]= useState([]);
     const [values, setValues] = useState({
         username:'',
         password:'',
         checkbox: true,
-    })
-    let myRef=React.createRef()
+        country:'',
 
+    })
 
 
-const onChange =(e) =>{
+    const onChange =(e) =>{
+        const target = e.target;
+        const value = target.type === 'checkbox' ? target.checked : target.value;
+        setValues({...values,[e.target.name]: value})
 
-    // if (e.target.value.trim()) {
-    //     myRef.current.removeAttribute('disabled')
-    // }
+    }
 
-    const target = e.target;
-    const value = target.type === 'checkbox' ? target.checked : target.value;
-    setValues({...values,[e.target.name]: value})
+    useEffect( () => {
+        const getCountry = async ()=>{
+            const response = await fetch("https://restcountries.com/v2/all");
+            const data = await response.json();
+            setCountry(await data);
+            console.log(data);
+        }
+        getCountry();
+
+    },[]);
+
+    function changeCountry(event){
+        const requiredCountry= event.target.value;
+        values.country=requiredCountry
+        console.log(requiredCountry)
+        event.preventDefault();
```

Comment:
not sure that it should be here , if it is needed in most cases it goes on first line of method